zenodo/modules/stats/templates/aggregations/v2/aggr-record-download-v1.0.0.json

```diff
@@ -0,0 +1,82 @@
+{
+  "template": "stats-record-download-*",
+  "settings": {
+    "index": {
+      "refresh_interval": "5s"
+    }
+  },
+  "mappings": {
+    "_default_": {
+      "dynamic_templates": [
+        {
+          "date_fields": {
+            "match_mapping_type": "date",
+            "mapping": {
+              "type": "date",
+              "format": "strict_date_hour_minute_second"
+            }
+          }
+        }
+      ]
+    },
+    "record-download-aggregation-v1.0.0": {
+      "_source": {
+        "enabled": false
+      },
+      "_all": {
+        "enabled": false
+      },
+      "date_detection": false,
+      "numeric_detection": false,
+      "properties": {
+        "timestamp": {
+          "type": "date",
+          "format": "strict_date_hour_minute_second"
+        },
+        "count": {
+          "type": "integer",
+          "index": "not_analyzed"
+        },
+        "unique_count": {
+          "type": "integer",
+          "index": "not_analyzed"
+        },
+        "volume": {
+          "type": "double",
+          "index": "not_analyzed"
+        },
+        "bucket_id": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "record_id": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "recid": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "conceptrecid": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "doi": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "conceptdoi": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "communities": {
+          "type": "string",
+          "index": "not_analyzed"
+        }
```

Comment:
Also include a `is_parent` boolean field, which will be in the `copy_fields` config as `is_parent=lambda *_: True` only for the `all-versions` aggregations